microbit/index03.html

```diff
@@ -74,8 +74,9 @@ <h2>Co nam sie może przydać?</h2>
     <ul>
         <li><span class="code">from microbit import *</span></li>
         <li>Pętla nieskończona <span class="code">while True:</span></li>
-        <li>Czekanie <span class="code">sleep(xxx)</span>, gdzie xxx to czas w milisekundach</li>
-        <li>Wyświetlanie <span class="code">display.show('xxx')</span></li>
+        <li>Czekanie <span class="code">sleep(x)</span>, gdzie x to czas w milisekundach</li>
+        <li>Pętla od 1 do 9 <span class="code">for cyfra in range(1,10,1):</span></li>
```

Comment:
Chcesz zrobić pętle w pętli? Czy to aby na pewno łatwe do zrozumienia? Pierwsze zadanie miało być na przypomnienie tego co było ostatnio.

Reply:
Chciałem aby zrobili coś nowego i poznali pętle for.

Powiemy, że ta pętla jest opcjonalna i można też bez niej, ale będzie więcej pisania